client/src/services/courses.ts

```diff
@@ -13,7 +13,7 @@ export class CoursesService {
   }
 
   async createCourse(data: Partial<Course>) {
-    const result = await axios.post<CourseResponse>(`/api/course/`, data);
+    const result = await axios.post<CourseResponse>(`/api/v2/courses/`, data);
```

Comment:
please run `npm run openapi` in `./nestjs` directory and it will update `CoursesApi`. After that, you can use ` this.coursesApi.createCourse(...)`